### PR TITLE
Adds Gondoland and the rare Gondolas

### PR DESCRIPTION
**Pull request by Militaires**
:cl: Basilman
add: A strange asteroid has drifted nearby...
/:cl:

A strange race of bear-like beings consisting of a head placed DIRECTLY on a pair of legs, their fur sells **for extremely high prices.**

The gondoland asteroid is the last natural habitat for these rare creatures.

image
image
image
image

### CODE REVIEW

**Comment by SyruseEln** [comment]
This is the single greatest PR I have yet seen.

**Comment by KingPhilipIII** [comment]
Glorious.

**Comment by BlueNothing** [comment]
What even are these?

File: code/modules/cargo/exports/sheets.dm [hunk 1]
```diff
@@ -57,6 +57,12 @@
 	unit_name = "lizard hide"
 	export_types = list(/obj/item/stack/sheet/animalhide/lizard)
 
+// Gondola hide. Mindbogglingly expensive.
+/datum/export/stack/skin/lizard
```
Comment (TrustyGun) [inline code/modules/cargo/exports/sheets.dm]: /datum/export/stack/skin/gondola

Comment (Militaires) [reply code/modules/cargo/exports/sheets.dm]: shit i forgot

**Comment by Militaires** [comment]
@KingPhilipIII that's in the DLC that comes with gondola variance when this gets merged
@bluenothing gondolas

**Comment by DoctoreMad** [comment]
Needs hillariously high health amount so you can contemplate your sins while you kill them.

**Comment by Anonmare** [comment]
tumblr_nvfymm03c81ug0kgwo1_500

**Comment by pubby** [comment]
It reminds me of Star Control 2, and for that reason I like it.

**Comment by FantasticFwoosh** [comment]
Honestly what were you smoking when you made this.

Random things to kill incased in asteroids is a welcome sight though for the desolate space-miner floating through space aided with a KA.

**Comment by DaxDupont** [comment]
@FantasticFwoosh whatever he was smoking, its some good shit.

**Comment by Militaires** [comment]
:sunglasses:

**Comment by PraiseRatvar** [comment]
ssssssssssssssssssperdooooo!!!

**Comment by Militaires** [comment]
gondola > spurdo

**Comment by Militaires** [comment]
hey guys is the gondola better like this with a bigger "moustache"

**Comment by Ratvar** [comment]
Give tnem slightly random leg lenth.

**Comment by optimumtact** [comment]
https://www.oranges.net.nz/~oranges/webm/contemplationdola.webm

**Comment by JJRcop** [comment]
🇫🇮

**Comment by Ausops** [comment]
wat dhe fug :-DDD

**Comment by KorPhaeron** [comment]
18 hours till I merge this

**Comment by XDTM** [comment]
Aw i thought you meant those boats they have in Venice

**Comment by Grazyn** [comment]
>https://www.oranges.net.nz/~oranges/webm/contemplationdola.webm

can we have this as background ambience in gondoland please

File: _maps/RandomRuins/SpaceRuins/gondolaasteroid.dmm [hunk 2]
```diff
@@ -0,0 +1,59 @@
+"a" = (/turf/template_noop,/area/template_noop)
```
Comment (Cyberboss) [inline _maps/RandomRuins/SpaceRuins/gondolaasteroid.dmm]: tgm

File: code/modules/mob/living/simple_animal/friendly/gondola.dm [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+/mob/living/simple_animal/pet/gondola
+	name = "gondola"
+	real_name = "gondola"
+	desc = "Gondola is the silent walker. Having no hands he embodies the Taoist principle of wu-wei (non-action) while his smiling facial expression shows his utter and complete acceptance of the world as it is. Its hide is extremely valuable."
+	response_help  = "pets"
+	response_disarm = "bops"
+	response_harm   = "kicks"
+	emote_see = list("watches.", "stares off into the distance.","contemplates.")
+	faction = list("gondola")
+	see_in_dark = 5
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/friendly/gondola.dm]: this should be a define

File: code/modules/mob/living/simple_animal/friendly/gondola.dm [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+	icon_living = "gondola"
+	icon_dead = "gondola_dead"
+	butcher_results = list(/obj/item/stack/sheet/animalhide/gondola = 1)
+	//Gondolas aren't affected by cold.
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+
+/mob/living/simple_animal/pet/gondola/IsVocal() //Gondolas are the silent walker.
+	return 0
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/friendly/gondola.dm]: FALSE

File: code/modules/mob/living/simple_animal/friendly/gondola.dm [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+	name = "gondola"
+	real_name = "gondola"
+	desc = "Gondola is the silent walker. Having no hands he embodies the Taoist principle of wu-wei (non-action) while his smiling facial expression shows his utter and complete acceptance of the world as it is. Its hide is extremely valuable."
+	response_help  = "pets"
+	response_disarm = "bops"
+	response_harm   = "kicks"
+	emote_see = list("watches.", "stares off into the distance.","contemplates.")
+	faction = list("gondola")
+	see_in_dark = 5
+	speak_chance = 1
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/friendly/gondola.dm]: Why is this set to 1 if IsVocal() returns 0

**Comment by Bawhoppen** [comment]
If you're going to make memes, can you atleast give this fucking space ruin some unique assets rather than reusing the same fucking sprites and same trees and structures and plants OVER AND FUCKING OVER YOU GUYS LITERALLY ALWAYS HALFASS EVERYTHING AND EVERYTHING FUCKING LOOKS COMPLETELY FUCKING SHITTY AND UNINSPIRED HOLY FUCK

**Comment by Bawhoppen** [comment]
Hello good sir, could I politely request you to help make your addition to the game more imaginative by adding new and unique assets, rather than reusing our existing ones? Doing this will help make the game universe feel more diverse, and will create a better sense of immersion, which is something we've been lacking lately. I'd like to broaden this comment to everyone, in hopes you will all take my advice on this matter.

**Comment by Bawhoppen** [comment]
I'm serious those please no more

**Comment by meeeeme** [comment]
simbly ebin :DDDDDD

**Comment by Militaires** [comment]
@cyberboss i added gondola hide there

**Comment by DoctoreMad** [comment]
I'll probably get rejected again, but can you consider adding more health than what /mob/living/simple_animal/pet gets? 

painandsuffering

**Comment by Militaires** [comment]
@doctoremad okay.

**Comment by Militaires** [comment]
actually ill give them as much health as a giant spider, which still isnt that much

**Comment by DoctoreMad** [comment]
@Cyberboss That thing about see_in_dark being a define is shocking because cats have "see_in_dark = 6" and dogs have "see_in_dark = 5" and actually that's what most mobs with see_in_dark are doing: https://github.com/tgstation/tgstation/blob/master/code/modules/mob/living/simple_animal/friendly

Of course without testing I'm not sure if I should make a issue about mobs using defines as vars or if I should call you out on this thread so I'll just do what takes less space.

**Comment by Cyberboss** [comment]
I may be full of shit, I expected see_in_dark to have defines in
https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/sight.dm
but they don't.

They should though

**Comment by KorPhaeron** [comment]
@Cyberboss The number is the number of tiles they see in the dark though, isn't it? It'd be like having a define for weapon forces

**Comment by KorPhaeron** [comment]
6 hours left

**Comment by DoctoreMad** [comment]
"damage = YES" sounds like the mark of a solid weapon to me.

**Comment by jughu** [comment]
add a shiny gondola 1% chance to spawn

**Comment by p440** [comment]
[https://www.youtube.com/watch?v=EJtfLJEA1iY](https://www.youtube.com/watch?v=EJtfLJEA1iY)

**Comment by AnturK** [comment]
I given up hope on preventing meme bloat what should just be varedits, but at least don't fill already bad space ruin list.

**Comment by Militaires** [comment]
by @FuryMcFlurry:


EBIN

**Comment by DoctoreMad** [comment]
Front sprite needs bigger beard you Furry McFurry

**Comment by FuryMcFlurry** [comment]
fuck

**Comment by Militaires** [comment]
@korphaeron the time isw now my friend

**Comment by Militaires** [comment]
best pr tbh

**Comment by Militaires** [comment]
woops

**Comment by DaxDupont** [comment]
@Militaires I saw the closed PR message in my email alerts I went NOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Comment by Militaires** [comment]
pranked

**Comment by Bawhoppen** [comment]
Can you please take my request?

**Comment by QualityVan** [comment]
:-DDDDDDDDD

**Comment by DoctoreMad** [comment]
2 days ago.

**Comment by PraiseRatvar** [comment]
@KorPhaeron what doctore said

**Comment by ExcessiveUseOfCobblestone** [comment]
Killing one should spawn GONDZILLA who is just a poorly resized version who chases you across time for your transgressions.

**Comment by Militaires** [comment]
SPURDO PRIDE SPARDE WIDE

**Pull request merged**